schemas/rum/error-schema.json

```diff
@@ -47,6 +47,37 @@
               "description": "Stacktrace of the error",
               "readOnly": false
             },
+            "causes": {
+              "type": "array",
+              "description": "Causes of the error",
+              "readOnly": false,
+              "items": {
+                "type": "object",
+                "description": "Properties for one of the error causes",
+                "required": ["message"],
+                "properties": {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should not we have most of the other error properties?